bin/src/explain.rs

```diff
@@ -19,7 +19,7 @@ pub mod main {
 
     pub fn main(explain_config: ExplainConfig) -> Result<(), StatixErr> {
         let explanation = super::explain(explain_config.target)?;
-        println!("{}", explanation);
+        println!("{explanation}");
```

Comment:
I started reviewing and quickly ran across this. This prompted me to enable a clippy lint that enforces this among most of the `clippy::pedantic` lints in #123. While that PR is split into many commits, they are between tiny and small and even though some conflict with this PR, the conflicts should be minimal. I hope. And, again, I'm not just putting this on you. You did the hard work already. So if you want to wait, I will rebase this for you myself, but that'll probably resume in a week. Sorry and thanks again.